db/seeds.rb

```diff
@@ -7,3 +7,8 @@
 FactoryBot.create_list(:company, 50)
 
 FactoryBot.create_list(:billing_contract, 100)
+
+Company.all.each do |company| 
```

Comment:
seed addresses for osbb and users as well

Reply:
done